libs/application/templates/financial-aid/src/lib/FinancialAidTemplate.ts

```diff
@@ -127,6 +130,33 @@ const FinancialAidTemplate: ApplicationTemplate<
           ],
         },
       },
+      [ApplicationStates.MUNCIPALITYNOTREGISTERED]: {
+        meta: {
+          name: application.name.defaultMessage,
+          lifecycle: {
+            shouldBeListed: false,
+            shouldBePruned: true,
+            whenToPrune: ONE_DAY,
+          },
+          roles: [
+            {
+              id: Roles.APPLICANT,
+              formLoader: () =>
+                import('../forms/MuncipalityNotRegistered').then((module) =>
+                  Promise.resolve(module.MuncipalityNotRegistered),
+                ),
+              write: {
+                answers: ['approveExternalData'],
```

Comment:
Do we need the write permissions in this state ApplicationStates.MUNCIPALITYNOTREGISTERED?